### DISCUSSION TITLE
What to modify when copying drawer components for custom interface

### DISCUSSION THREAD

**Discussion by jsx101** [directus/directus#17301] [Archive]
I would like to copy and alter the DrawerCollection and DrawerItem components for a custom interface I'm working on.

Has anyone copy-pasted these components for their custom interfaces before? It can quickly snowball if I have to copy the Vue files for the components and all imported files, and the imports for those imported files, and so on. I'm looking for the modifications you made to the private imported files, or which public packages you used to replace the private ones.

I feel like I'm cheating here but the last time I tried it, I got everything cleaned up but the drawer still wouldn't work. I wasted a lot of time and got nothing.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/17234

**Reply by jsx101** [reply]
Thanks! I missed that discussion.
